ambari-server/src/main/resources/common-services/AMBARI_METRICS/0.1.0/package/files/grafana-dashboards/HDP/grafana-hdfs-datanodes.json

```diff
@@ -83,19 +83,18 @@
           "targets": [
             {
               "aggregator": "avg",
-              "app": "datanode",
+              "app": "namenode",
```

Comment:
I guess all the "app" fields should be "datanode" on this dashboard, aren't they ?